master/docs/manual/configuration/wwwhooks.rst

```diff
@@ -140,8 +140,8 @@ The GitHub hook has the following parameters:
 
 ``token``
     If your GitHub or GitHub Enterprise instance does not allow anonymous communication, you need to provide an access token.
-    Instructions can be found `here <https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/>`_.
-
+    Instructions can be found `here <https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/>`_. This attribute is rendered using the :class:`~buildbot.interfaces.IRenderable` interface, the only property available is ``full_name``, of the format ``{owner}/{full_name}``.
```

Comment:
Please split into multiple lines.

Reply:
at what precise column should I split?

Reply:
Most of the documentation uses "one line per sentence style". I've found it hard to work with because documentation even in rst form is mostly read, not written. So I suggest to break at 100 column, like the rest of Buildbot code.